app/models/ingest_job.rb

```diff
@@ -14,7 +14,7 @@ class IngestJob
 
   # valid ingest actions to perform
   VALID_ACTIONS = %i[
-    ingest_expression ingest_cluster ingest_cell_metadata subsample differential_expression render_expression_arrays
+    ingest_expression ingest_cluster ingest_cell_metadata subsample differential_expression render_expression_arrays ingest_anndata
```

Comment:
Keeping more similar things closer to each other, for coherence:

```suggestion
    ingest_expression ingest_cluster ingest_cell_metadata ingest_anndata subsample differential_expression render_expression_arrays
```